src/pip/_internal/network/utils.py

```diff
@@ -44,11 +44,11 @@ def raise_for_status(resp):
         reason = resp.reason
 
     if 400 <= resp.status_code < 500:
-        http_error_msg = u'%s Client Error: %s for url: %s' % (
+        http_error_msg = '%s Client Error: %s for url: %s' % (
             resp.status_code, reason, resp.url)
 
     elif 500 <= resp.status_code < 600:
-        http_error_msg = u'%s Server Error: %s for url: %s' % (
+        http_error_msg = '%s Server Error: %s for url: %s' % (
```

Comment:
Might be a good chance to turn these into f-strings.

Reply:
Moving to f-strings is a good idea. I was planning to handle that as a separate PR as there will be lots to fix and I didn't want to muddle the focus of other PRs. Does that approach work for you?

Reply:
I prefer a separate PR, yea.